lib/easy-stackup.js

```diff
@@ -0,0 +1,72 @@
+//simpler API main function
+'use strict'
+var gerberToSvg = require('gerber-to-svg')
+var shortId = require('shortid')
+var whatsThatGerber = require('whats-that-gerber')
+
+var pcbStackup = require('./index')
+
+module.exports = function(layers, optionsOrCallback, callback) {
+  var options = {}
+
+  if (typeof optionsOrCallback === 'object') {
+    options = optionsOrCallback
+  }
+  else if (typeof optionsOrCallback === 'function') {
+    callback = optionsOrCallback
+  }
+
+  if (!('id' in options)) {
+    options.id = shortId.generate()
+  }
+
+  var layerCount = layers.length
+  var stackupLayers = []
+
+  var finishLayer = function() {
+    if (layerCount <= 1) {
+      var stackup = pcbStackup(stackupLayers, options)
+      stackup.layers = stackupLayers.map(function(layer) {
+        return {
+          layerType: layer.type.id,
+          gerber: layer.converter,
+          options: layer.options
+        }
+      })
+      callback(null, stackup)
+    }
+    layerCount -= 1
+  }
+
+  if (layerCount === 0) {
+    finishLayer()
+  }
+  else {
+    for (var i in layers) {
```

Comment:
Generally prefer `array.forEach`, and additionally, don't use for..in loops with arrays because [iteration order isn't necessarily guaranteed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)